### PR TITLE
Add web5/crypto to docs ci check

### PR DESCRIPTION
**Pull request by leordev**
wip.. we need to fix all the docs errors

### CODE REVIEW

File: packages/crypto/src/index.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 export * from './jose.js';
 export * from './local-kms-crypto.js';
-export * as utils from './utils.js';
+import * as utils from './utils.js';
+export { utils };
```
Comment (frankhinek) [inline packages/crypto/src/index.ts:4]: can you switch this back?  there's nothing wrong with this syntax

Comment (leordev) [reply packages/crypto/src/index.ts:4]: I can't, unfortunately it's a constraint with api-extractor...

Comment (leordev) [reply packages/crypto/src/index.ts:4]: https://github.com/microsoft/rushstack/issues/2780

**Comment by leordev** [comment]
Closing in favor of #402

**Comment by leordev** [comment]
Closing in favor of #402

**Pull request closed without merging**